### PR TITLE
69 new vox phrases

### PR DESCRIPTION
**Pull request by mc-oofert**
# attempt 2 of #80569 (no rizz, gyatt, or mothblocks this time)
# created via https://github.com/AlexMorgan3817/ss13-vox

## About The Pull Request

69 or so roughly new phrases
words of note: felinid, jolly, christmas, lightbulb, present, presents, lizard, lizardperson, never, thanks, while, was, okay, had, her, some numbers, ever
wordlist used (duplicates inside:)
[common.txt](https://github.com/tgstation/tgstation/files/13765603/common.txt)

https://github.com/tgstation/tgstation/assets/70376633/adaa1607-8c18-4fed-a6e8-1b7ac0db3303

https://github.com/tgstation/tgstation/assets/70376633/ffbd1b1d-6b2c-4921-ae77-45ea4056348f

## Why It's Good For The Game

vox shenanigans

## technobabble
anyway so getting the shitty project running took me 80 hours or so over many attempts but
the linked fork has less of that poetry python bullshit
just dont install python3.6 its pointless, install python3
anyway so run sudo python3 setup.py
follow instructions till it actually works
generate.sh will not work well so just run "sudo python3 create.py" directly
also you could just skip the sudo part by just running sudo -i before all that
create.py will create both voices for vox_masc and vox_fem for some reason but you could avoid that by removing masc or whatever the fuck the sex is called from create.py to save some time by doing fem solely
also you should set codebase to tg in codebase.yaml and replace tglist.jinja2 in templates with this
```
GLOBAL_LIST_INIT(vox_sounds, list(
  {%- for sex, phrases in SEXES.items() %}
    {%- for phrase in phrases %}
"{{- phrase.id -}}" = 'sound/vox_fem/{{- phrase.id -}}.ogg',
    {%- endfor %}
  {%- endfor %}
))
```
which should create roughly correct code for tg but you still need to do some text work
like removing duplicates n stuff and moving sounds from dist/sounds to the actual proper sound folder

anyway thank you for coming to my rant

## Changelog
:cl:
add: 69(roughly?) new vox phrases
/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
- Why "maintainer"?
- Why "sockmuncher" and "taildragger"?
- Just a note but rather than including "lizardperson" as its own, you could include "person" so one can just do "lizard person". (Same for "moth person"

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 1]
```diff
@@ -31,10 +31,7 @@ GLOBAL_LIST_INIT(vox_sounds, list("abduction" = 'sound/vox_fem/abduction.ogg',
 "aft" = 'sound/vox_fem/aft.ogg',
 "agent" = 'sound/vox_fem/agent.ogg',
 "ai" = 'sound/vox_fem/ai.ogg',
-"airlock" = 'sound/vox_fem/airlock.ogg',
 "air" = 'sound/vox_fem/air.ogg',
-"alarm" = 'sound/vox_fem/alarm.ogg',
-"alert" = 'sound/vox_fem/alert.ogg',
```
Comment (MrMelbert) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm:37]: Can you put these back in ABC order? Should be easy to do it automatically in VScode

**Comment by iwishforducks** [comment]
sockmuncher and taildragger is amazing. after all, ais ARE supposed to be asimov and racist. maintainer is funny too. dont let the fun police ruin this pr 😅

**Comment by theselfish** [comment]
nice

**Comment by MrMelbert** [comment]
> sockmuncher and taildragger are amazing. after all, ais ARE supposed to be asimov and racist. maintainer is funny too. dont let the fun police ruin this pr 😅

i'm not fun policing i'm  just genuinely confused where they're coming from

**Comment by mc-oofert** [comment]
> * Why "maintainer"?

Chief Engineer , maintainer of the supermatter shonk

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 1 above)
Comment (mc-oofert) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm:37]: i code in notepad++ bro

Comment (mc-oofert) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: did

**Comment by mc-oofert** [comment]
>     * Why "sockmuncher" and "taildragger"?

alternate word for mothpeople and lizardpeople respectively

**Comment by iwishforducks** [comment]
> > sockmuncher and taildragger are amazing. after all, ais ARE supposed to be asimov and racist. maintainer is funny too. dont let the fun police ruin this pr 😅
> 
> i'm not fun policing i'm just genuinely confused where they're coming from

sorry, i was absolutely jaded and took ur innocent questions as passive aggressive rather than… simply what they were. just questions. ur good. im bad AND wrong

**Comment by DATA-xPUNGED** [comment]
Can we put Morgue in this

**Comment by Admiral-Gigaton** [comment]
Add "tax" so I can say "the tax man is here"

**Review by Ghommie — APPROVED** [review]
Head and tails, this PR roughly 500 kilobytes of sound resources.

**Comment by mc-oofert** [comment]
> Can you please indent this? It's impossible to read the diffs for downstreams, and just to see what was changed in general.

it was like this already and downstream is out of my concerns

**Pull request merged**